### ISSUE TITLE
terraformls - need terraform binary inside the image

### ISSUE THREAD

**Issue by dbgeek** [lspcontainers/dockerfiles#25]
Formatting terraform/hcl code does not work. It needs the terraform binary install in the image. See the error message below.

How should we keep track of the two different versions(terraformls, terraform)?

Error message in the lsp.log
```
[ ERROR ] 2021-05-17T16:10:10+0200 ] ...im/HEAD-e0a01bd_2/share/nvim/runtime/lua/vim/lsp/rpc.lua:457 ]  "rpc"   "docker"      "stderr" '2021/05/17 14:10:10 rpc_logger.go:45: Error for "textDocument/formatting" (ID 2): [-32098] Terraform (CLI) is required. Please install Terraform or make it available in $PATH\n2021/05/17 14:10:10 server.go:247: Completed 1 requests [196.468µs elapsed]\n'
```

**Comment by dbgeek** [comment]
Maybe we can modify the data structure under [include](https://github.com/lspcontainers/dockerfiles/blob/main/.github/workflows/build-image.yml#L17) 

from 
```
- context: bashls            
   tag: bash-language-server
   version: 1.17.0
```

to
```
- context: bashls            
   tag: bash-language-server
   build-args: |
     version: 1.17.0
```

example  for terraformls
to
```
- context: bashls            
   tag: bash-language-server
   build-args: |
     version: 1.17.0
     terraform-version: 0.15.3
```

Then we can maybe do like this in `docker/build-push-action@v2`

```
build-args: ${{matric.build-args}}
```

If this works we can add `terraform-version` argument to terraformls Dockerfile?

**Comment by erkrnt** [comment]
This has been merged from #26
